homeassistant/components/mealie/const.py

```diff
@@ -12,3 +14,11 @@
 ATTR_RECIPE_ID = "recipe_id"
 ATTR_URL = "url"
 ATTR_INCLUDE_TAGS = "include_tags"
+
+MIN_REQUIRED_MEALIE_V = AwesomeVersion("v1.0.0")
```

Comment:
It is better to be descriptive.

```suggestion
MIN_REQUIRED_MEALIE_VERSION = AwesomeVersion("v1.0.0")
```